library/classes/Installer.class.php

```diff
@@ -1017,7 +1017,7 @@ public function install_gacl()
         // xl('Things that front office can read and partly modify')
         $gacl->add_acl(
             array(
-                'patients' => array('appt', 'demo', 'trans', 'notes'),
+                'patients' => array('appt', 'demo'),
```

Comment:
note these changes are in the write acl and not the wsome acl

Reply:
nice spot!